.github/actions/run/action.yml

```diff
@@ -47,6 +47,15 @@ runs:
         path: ~/.m2/repository
         key: ${{ runner.os }}-maven-agent-test-run-${{ hashFiles('**/pom.xml') }}
         restore-keys: ${{ runner.os }}-maven-agent-test-run-
+    - name: Install docker-compose
+      shell: bash
+      if: runner.os != 'Windows'
+      run: |
+        if ! command docker-compose 2>&1 > /dev/null; then
+          echo "Installing docker-compose"
+          sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
+        fi
```

Comment:
@kezhenxu94 Is this a good practice?

Reply:
This is not a good choice, we did this because our infra e2e uses the docker-compose in third party SDK (testcontainers), that we cannot easily fix. Please fix as https://github.com/apache/skywalking-java/pull/716#issuecomment-2349282689

Reply:
OK, I will revert this change.